stack-14.yaml

```
@@ -1,4 +1,4 @@
-resolver: lts-14.12
+resolver: lts-14.27
 packages:
```

From Travis, you'll need `allow-newer: true` here as well
```suggestion
allow-newer: true
packages:
```